test/image/mocks/scatter_fill_gradient_tonext.json

```diff
@@ -0,0 +1,42 @@
+{
```

Comment:
Thanks very much for the excellent PR :1st_place_medal: 
We are experiencing a bug in our parallel runs on CircleCI.
Please rename this mock as well as the baseline to start with `zz-` i.e. 
`zz-` like `zz-scatter_fill_gradient_tonext.json` and
`zz-` like `zz-scatter_fill_gradient_tonext.png`.
This would simply help putting them at the end of the list and make CircleCI image tests green.
Thank you!